src/guacd-docker/bin/build-libvncserver.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh -e
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+##
+## @fn build-libvncserver.sh
+##
+## Builds the source of libvncserver
+##
+## @param BUILD_DIR
+##     The working directory to download, extract and build the libvncserver distribution
```

Comment:
Same here - missing period. Please also wrap your lines at 80 columns where possible (and where doing so doesn't hurt readability). It's sometimes a tough call whether to wrap a line of code, but there are almost no cases where doing so would be bad for the documentation.

See the style of: https://github.com/apache/incubator-guacamole-server/blob/6236eb8f98ff96c495ae94cb4fe3279c202db465/src/guacd-docker/bin/build-guacd.sh#L27-L29